app/views/relationships/index.html.erb

```diff
@@ -0,0 +1,21 @@
+<% content_for :title do %>
+  Relationships
+<% end %>
+<h1>Relationships</h1>
+<%- if current_user && current_user.admin?  %>
+  <%= link_to "Create Relationship", new_relationship_path, {:class=>'btn btn-default'} %>
```

Comment:
Remember to eliminate this and replace it with some helpful text directing the user to create a relationship from one of the terms.